pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -795,6 +803,14 @@ func (o *MustGatherOptions) newPod(node, image string, hasMaster bool) *corev1.P
 					// always force disk flush to ensure that all data gathered is accessible in the copy container
 					Command: []string{"/bin/bash", "-c", fmt.Sprintf("%s & %s; sync", volumeUsageChecker, executedCommand)},
 					Env: []corev1.EnvVar{
+						{
+							Name:  "MUST_GATHER_SINCE",
```

Comment:
nit: I think, it is better to not set these environment variables, if since or since time are not set.

Reply:
done!